web/app/dashboard/add-project-dialog/add-project-dialog.component.ts

```diff
@@ -1,12 +1,71 @@
-import {Component, OnInit} from '@angular/core';
+import {Component, Inject, OnInit} from '@angular/core';
+import {MAT_DIALOG_DATA} from '@angular/material';
+import {Observable} from 'rxjs/Observable';
+
+import {Repository} from '../../models/repository';
+import {AddProjectRequest} from '../../services/data.service';
+
+export interface AddProjectDialogConfig {
+  repositories: Observable<Repository[]>;
+}
+
+interface TriggerOption {
+  viewValue: string;
+  value: 'commit'|'nightly';
+}
+
+interface TimeSelectorData {
+  hour: number;
+  isAm: boolean;
+}
+
+function timeSelectDataToMilitaryTime(timeData: TimeSelectorData): number {
```

Comment:
Is this being decided based on the user agent?

Reply:
I'm not sure i understand the question. Could you expand on this?

Currently (i believe) our scheduled time is Hour+Minutes. So i believe we need 24 hour time format to work. This is why we do this conversation. So 12AM is 0, and 1pm is 13.

Reply:
Ahhh alright, sorry I misunderstood that code then